src/DataBox/DataBox/Help/Get-AzDataBoxCredential.md

```diff
@@ -0,0 +1,132 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.DataBox.dll-Help.xml
+Module Name: Az.DataBox
+online version:
```

Comment:
You will need to fill this out for each of these files.  https://docs.microsoft.com/en-us/powershell/module/az.databox/get-azdataboxcredential